.buildkite/pipeline.yml

```diff
@@ -48,8 +49,20 @@ steps:
         allow_failure: false
       - step: unit-tests-linux
         allow_failure: false
+
+  - label: ":windows: Run stack tests"
+    key: stack-tests-windows
+    command: ".buildkite/scripts/stack_tests_windows.ps1"
+    agents:
+      provider: "gcp"
+      image: "${WINDOWS_AGENT_IMAGE}"
+    depends_on:
+      - step: check-static
+        allow_failure: false
       - step: unit-tests-windows
         allow_failure: false
+    artifact_paths:
+      - 'C:\ProgramData\chocolatey\logs\chocolatey.log'
```

Comment:
Is this needed? Could it be removed?

Reply:
I found it useful to see what chocolatey is doing, so I left it. But I can remove it and we can add it back when debugging issues installing things, as you prefer.

Reply:
Ok, if it has been helpful, let's keep it.